src/main/java/com/mael/payroll/cards/TimeCard.java

```diff
@@ -0,0 +1,19 @@
+package com.mael.payroll.cards;
+
+public class TimeCard {
+    private int date;
+    private double hours;
+
+    public TimeCard(int date, double hours) {
```

Comment:
I didn't need to deal with dates up until now, at the final stage, where I have to calculate the pay and pay only if it's pay day. Different types of payment have different pay dates.

I didn't know about `LocalDate`, gonna check it ou! :+1: